### PR TITLE
Security Compliance Toolkit: 2018 missing in the version matrix

### PR DESCRIPTION
**Pull request by RAJU2529**
it is my own PR , after verifying concerned website links, I added the names with corresponding links

### CODE REVIEW

**Comment by illfated** [comment]
Based on my general GitHub experience, I would like to suggest an even more precise PR title text:

**Security Compliance Toolkit: version matrix updated for 2018**

**Comment by illfated** [comment]
(Ref. #6288 )

**Comment by RAJU2529** [comment]
@illfated . Title is changed

**Comment by illfated** [comment]
@martyav : Thank you for the practical feedback.

@RAJU2529 : Will you be able to use the advised changes on your own, or do you prefer suggestions in the source text, for you to commit?

**Comment by RAJU2529** [comment]
@illfated . How do edit that page again'

**Comment by illfated** [comment]
That should be fairly easy, by going to your page
https://github.com/RAJU2529/windows-itpro-docs/blob/a04e5f4be6b59b1a4db9dc9d0919f45e4b303efe/windows/security/threat-protection/windows-security-configuration-framework/get-support-for-security-baselines.md
and edit by using the pen symbol:

[edit_file](https://github.com/RAJU2529/windows-itpro-docs/blob/a04e5f4be6b59b1a4db9dc9d0919f45e4b303efe/windows/security/threat-protection/windows-security-configuration-framework/get-support-for-security-baselines.md)

**Comment by RAJU2529** [comment]
@illfated . Sorry i am getting below error message, i could not able to click on edit button
**You must be on a branch to make or propose changes to this file**

**Comment by RAJU2529** [comment]
@illfated . please see below the post from another user. I need your opinion
**https://answers.microsoft.com/en-us/windows/forum/all/reset-windows-10-system-or-local-account-password/40533db8-d707-4637-bc2a-cdf3db60fb88**

i answered for that question, i said there is a way to reset the forgot windows or local account without without reset or without reinstall windows .

i think you know about all account password is stoted by regsitry in location of C:\windows\system32\config\software  file.
i have one file ,that file is able to unlock that account to a blank password.

But the forum moderator is deleted my answer , because violating of security settings.

**Comment by illfated** [comment]
> @illfated . Sorry i am getting below error message, i could not able to click on edit button
> **You must be on a branch to make or propose changes to this file**

OK, I also get that error message, but that is because I don't own your page. Never mind, I will suggest the changes directly in the PR code instead.

**Comment by illfated** [comment]
> @illfated . please see below the post from another user. I need your opinion
> **https://answers.microsoft.com/en-us/windows/forum/all/reset-windows-10-system-or-local-account-password/40533db8-d707-4637-bc2a-cdf3db60fb88**
> 
> i answered for that question, i said there is a way to reset the forgot windows or local account without without reset or without reinstall windows .
> 
> i think you know about all account password is stoted by regsitry in location of C:\windows\system32\config\software file.
> i have one file ,that file is able to unlock that account to a blank password.
> 
> But the forum moderator is deleted my answer , because violating of security settings.

Well, I guess that solution is considered too "hacky" and not the correct way to perform Windows OS repair or maintenance in Safe Mode. Even though some methods are possible, like the one you suggested, I presume it will never be considered an Enterprise solution, but rather a "1 computer at a time hack" to salvage user data. Anyway, that also means your method is not supported by Microsoft.

**Comment by RAJU2529** [comment]
@illfated .
Microsoft is the recommended official tool that contains reset the local password in that name of the tool kit is  DART for windows 10.
I have that tool in iso, i kept it for personal use.
here is page 
**https://docs.microsoft.com/en-us/microsoft-desktop-optimization-pack/dart-v10/getting-started-with-dart-10**

I downloaded that iso  September  2016

**Comment by illfated** [comment]
Well, maybe you were using a correct solution, but there is nothing I can do if the moderator of the support page refuses to accept your suggestion. I know it is quite extreme to delete one or more comments made in an effort to help others, but I can only suggest that you try a different page, or contact the user with the problem directly.

Anyway, I don't think we should discuss external issues here in your PR.

**Comment by RAJU2529** [comment]
@illfated. What about my existing Created  PR issue.

**Comment by illfated** [comment]
OK, sorry, I got distracted by corona news. I will update your PR.

File: windows/security/threat-protection/windows-security-configuration-framework/get-support-for-security-baselines.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ No. SCM supported only SCAP 1.0, which was not updated as SCAP evolved. The new
...
+|Windows 10 | [1809 (October 2018)](https://docs.microsoft.com/en-us/archive/blogs/secguide/security-baseline-draft-for-windows-10-v1809-and-windows-server-2019) <p>[1803 (RS4)](https://docs.microsoft.com/en-us/archive/blogs/secguide/security-baseline-for-windows-10-v1803-redstone-4-draft) <p>[1709 (RS3)](https://blogs.technet.microsoft.com/secguide/2017/09/27/security-baseline-for-windows-10-fall-creators-update-v1709-draft/) <p> [1703 (RS2)](https://blogs.technet.microsoft.com/secguide/2017/08/30/security-baseline-for-windows-10-creators-update-v1703-final/) <p>[1607 (RS1)](https://blogs.technet.microsoft.com/secguide/2016/10/17/security-baseline-for-windows-10-v1607-anniversary-edition-and-windows-server-2016/) <p>[1511 (TH2)](https://blogs.technet.microsoft.com/secguide/2016/01/22/security-baseline-for-windows-10-v1511-threshold-2-final/) <p>[1507 (TH1)](https://blogs.technet.microsoft.com/secguide/2016/01/22/security-baseline-for-windows-10-v1507-build-10240-th1-ltsb-update/)| October 2018 <p>March 2018 <p>October 2017 <p>August 2017 <p>October 2016 <p>January 2016<p> January 2016 |[SCT 1.0](https://www.microsoft.com/download/details.aspx?id=55319) |
```
Comment (illfated) [inline windows/security/threat-protection/windows-security-configuration-framework/get-support-for-security-baselines.md]:
```suggestion
| Windows 10 | [1809 (October 2018)](https://docs.microsoft.com/archive/blogs/secguide/security-baseline-draft-for-windows-10-v1809-and-windows-server-2019) <br>[1803 (RS4)](https://docs.microsoft.com/archive/blogs/secguide/security-baseline-for-windows-10-v1803-redstone-4-draft) <br>[1709 (RS3)](https://blogs.technet.microsoft.com/secguide/2017/09/27/security-baseline-for-windows-10-fall-creators-update-v1709-draft/) <br> [1703 (RS2)](https://blogs.technet.microsoft.com/secguide/2017/08/30/security-baseline-for-windows-10-creators-update-v1703-final/) <br>[1607 (RS1)](https://blogs.technet.microsoft.com/secguide/2016/10/17/security-baseline-for-windows-10-v1607-anniversary-edition-and-windows-server-2016/) <br>[1511 (TH2)](https://blogs.technet.microsoft.com/secguide/2016/01/22/security-baseline-for-windows-10-v1511-threshold-2-final/) <br>[1507 (TH1)](https://blogs.technet.microsoft.com/secguide/2016/01/22/security-baseline-for-windows-10-v1507-build-10240-th1-ltsb-update/)| October 2018 <br>March 2018 <br>October 2017 <br>August 2017 <br>October 2016 <br>January 2016<br> January 2016 |[SCT 1.0](https://www.microsoft.com/download/details.aspx?id=55319) |
```

File: windows/security/threat-protection/windows-security-configuration-framework/get-support-for-security-baselines.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ No. SCM supported only SCAP 1.0, which was not updated as SCAP evolved. The new
 
 | Name | Build | Baseline Release Date | Security Tools |
 |---|---|---|---|
```
Comment (illfated) [inline windows/security/threat-protection/windows-security-configuration-framework/get-support-for-security-baselines.md]:
```suggestion
| ---- | ----- | --------------------- | -------------- |
```
(readability alignment with the header text)

**Comment by illfated** [comment]
Rich text view of the result after my suggested changes:

| Name | Build | Baseline Release Date | Security Tools |
| ---- | ----- | --------------------- | -------------- |
| Windows 10 | [1809 (October 2018)](https://docs.microsoft.com/archive/blogs/secguide/security-baseline-draft-for-windows-10-v1809-and-windows-server-2019) <br>[1803 (RS4)](https://docs.microsoft.com/archive/blogs/secguide/security-baseline-for-windows-10-v1803-redstone-4-draft) <br>[1709 (RS3)](https://blogs.technet.microsoft.com/secguide/2017/09/27/security-baseline-for-windows-10-fall-creators-update-v1709-draft/) <br> [1703 (RS2)](https://blogs.technet.microsoft.com/secguide/2017/08/30/security-baseline-for-windows-10-creators-update-v1703-final/) <br>[1607 (RS1)](https://blogs.technet.microsoft.com/secguide/2016/10/17/security-baseline-for-windows-10-v1607-anniversary-edition-and-windows-server-2016/) <br>[1511 (TH2)](https://blogs.technet.microsoft.com/secguide/2016/01/22/security-baseline-for-windows-10-v1511-threshold-2-final/) <br>[1507 (TH1)](https://blogs.technet.microsoft.com/secguide/2016/01/22/security-baseline-for-windows-10-v1507-build-10240-th1-ltsb-update/)| October 2018 <br>March 2018 <br>October 2017 <br>August 2017 <br>October 2016 <br>January 2016<br> January 2016 |[SCT 1.0](https://www.microsoft.com/download/details.aspx?id=55319) |

**Comment by illfated** [comment]
Oh, and by the way: I finally managed to browse my way to the correct file path for the file version for you to edit (so we can remember this for later):
- https://github.com/RAJU2529/windows-itpro-docs/blob/patch-53/windows/security/threat-protection/windows-security-configuration-framework/get-support-for-security-baselines.md

Notice in particular the middle part of the path, **`/blob/patch-53/`**
between 
`https://github.com/RAJU2529/windows-itpro-docs` 
and 
`windows/security/threat-protection/windows-security-configuration-framework/get-support-for-security-baselines.md` -- the /blob/patch-53/ bit shows that you are in a _branch_ (patch-53).

I will make sure to keep this in mind for later.

**Pull request merged**